localization/english/event_localization/nickname/wc_nickname_events_l_english.yml

```
@@ -0,0 +1,17 @@
+﻿l_english:
+ WCNIC.0001.t:0 "The Oldest"
+ WCNIC.0001.desc:0 "As a long time has passed as the ruler your tribe, you notice your subjects and the surrounding population has started to refer to you as “the Old Murkeye” due to your long reign and the many years it has been since you have been a tadpole. The noblest of your subjects give the idea of a ceremony to earn this as an official nickname, they prepare the ceremony and all that is left is for you to step on the rock and give the gurgling speech for the army of Murloc raiders in front of you."
+ WCNIC.0001.a:0 "Mrghllghghllghg !"
+ WCNIC.0001.b:0 "Mrglrlhglrlg..."
+ WCNIC.0001.tt.a:0 "The murloc raiders cheer and gurgle your nickname."
+ WCNIC.0001.tt.b:0 "You turn away from the stone and walk away in refusal, eventually the Murloc crowd dissipates. The nickname of "the Old Murkeye" slowly fades away in time."
+
+ WCNIC.0002.t:0 "Oh, how the years have past"
+ WCNIC.0002.desc:0 "It has been many years since you have ascended to the throne of your country and many more since you have come of age, your council and most noble of courtiers have proposed a celebration of your long reign and wish for you to be known in the sagas as “the Old”, the ceremony for your new title is to be held right away. The population all come together to dine and praise your name."
+ WCNIC.0002.a:0 "I raise my glass to many more propitious years as their ruler."
+ WCNIC.0002.b:0 "Let's not be so pompous."
+
+ martial_strategy.1051.g.wc:0 "I can strike fear into my enemies with cold and heartless treatment."
+ martial_strategy.1051.h.wc:0 "My way of fighting with strength and pride. "
```

I'd suggest changing it as suggested below, that or change it to "My way of fighting is with strength and pride."
```suggestion
 martial_strategy.1051.h.wc:0 "I fight with strength and pride. "
```

---

Done in 68eb5459